Answers.md

```diff
@@ -2,10 +2,22 @@
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
-1. What does it mean to think in react?
+    It is a JS library for building user interfaces, It helps by seperating our reusuable components into their own component files.
 
-1. Describe state.
+2. What does it mean to think in react?
 
-1. Describe props.
+    "Can I seperate these into component files to make my code manageable"
 
-1. What are side effects, and how do you sync effects in a React component to state or prop changes?
+3. Describe state.
+
+    State is what we display in the components.

```

Comment:
state is a special kind of data within react.  When state is changed via hooks or class component functions (we will learn later), it forces a rerender of the app.  State can also be described as an object.  State can or cannot be used to display properties, often it is just used for logic